Lib/fontTools/feaLib/parser.py

```diff
@@ -77,7 +81,7 @@ def parse_anchor_(self):
                     'Unknown anchor "%s"' % name,
                     self.cur_token_location_)
             self.expect_symbol_(">")
-            return ast.Anchor(location, name, anchordef.x, anchordef.y,
+            return self.ast.Anchor(location, name, anchordef.x, anchordef.y,
                               anchordef.contourpoint,
```

Comment:
since you added extra characters at the beginning of this multi-line statement (`self.`), I suggest you keep the indentation of the subsequent lines in sync, so the code follows [PEP8] style.

https://www.python.org/dev/peps/pep-0008/#indentation